sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -19,78 +20,246 @@ export const FastFrameTemplate = html<FastFrame>`
                 <fast-tab id="swatches-tab" title="Styles">${SwatchesIcon}</fast-tab>
                 <fast-tab id="density-tab" title="Density">${ScreenIcon}</fast-tab>
                 <fast-tab-panel id="contrast-tab-panel">
-                    <fast-switch
-                        checked="${x => x.darkMode}"
-                        @change="${(x, c) => x.themeChange(c.event as MouseEvent)}"
-                    >
-                        Dark Mode
-                    </fast-switch>
+                    <div class="content">
+                        <h1><span class="content-heading-highlight">Fast Frame</span> Dark Mode</h4>
+                        <h2>Pre-built for both light and dark modes</h2>
+                        <p>
+                            Sem viverra fringilla at magna turpis in. Nullam adipiscing fusce auctor semper arcu felis. Purus et enim feugiat arcu. Lectus neque sem est ridiculus tempus urna.
+                        </p>
+                        <div class="content-control-container" >
+                            <label for="dark-mode-switch">Dark mode</label>
+                            <fast-switch
+                                id="dark-mode-switch"
+                                checked="${x => x.darkMode}"
```

Comment:
When setting checked, you generally want to set the checked _property_ and not the checked _attribute_. This is because the checked _attribute_ will only change the checked _property_ if the control is not in a dirty state. This works now but could cause unexpected behavior down the line.

```js
:checked="${x => x.darkMode}"
```